examples/buildpacks/build.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
 set -e
-images=$(echo $IMAGES | tr " " "\n")
+image=$(echo $IMAGE)
 
-for image in $images
-do
-    pack build $image
-    if $PUSH_IMAGE
-    then
-        docker push $image
-    fi
-done
+if [ !-z "$image" ]
+  pack build $image
```

Comment:
missing `then`.

Reply:
aah thanks!  i ran it  3 times thinking this is a flake.

Reply:
hmm there is also another flake i saw then "TestDevSync" which failed 3 times.